resources/locale/zh-TW.ini

```diff
@@ -28,7 +28,7 @@ CodingType.Description="編碼最終封包時所使用的編碼類型。\n- 「\
 MaximumReferenceFrames="最大參考訊框"
 MaximumReferenceFrames.Description="輸出檔案的最大參考訊框。這對資源這個功能的編碼器的畫質有絕對的影響。\n\n這個選項是靜態的，且不能在編碼時變更。"
 RateControlMethod="速率控制方法"
-RateControlMethod.Description="應使用何種方法以控制（位元）速率？\n- 「\@RateControlMethod.CQP\@」為每個訊框分配固定量化參數，建議用於高品質至接近無損的錄製。\n-「\@RateControlMethod.CBR\@」嘗試盡可能地接近 \@Bitrate.Target\@，可以選擇用 \@FillerData\@ 填充，建議用於串流。\n- 「\@RateControlMethod.VBR\@」嘗試盡可能地接近 \@Bitrate.Peak\@，若可能，則盡可能地降低至 \@Bitrate.Target\@，建議用於小尺寸錄製。\n- 「\@RateControlMethod.VBRLAT\@」與「@RateControlMethod.VBR\@」類似，但將編碼延遲列入考量。"
+RateControlMethod.Description="應使用何種方法以控制（位元）速率？\n- 「\@RateControlMethod.CQP\@」為每個訊框分配固定量化參數，建議用於高品質至接近無損的錄製。\n-「\@RateControlMethod.CBR\@」嘗試盡可能地接近 \@Bitrate.Target\@，可以選擇用 \@FillerData\@ 填充，建議用於串流。\n- 「\@RateControlMethod.VBR\@」嘗試盡可能地接近 \@Bitrate.Peak\@，若可能，則盡可能地降低至 \@Bitrate.Target\@，建議用於小尺寸錄製。\n- 「\@RateControlMethod.VBRLAT\@」與「\@RateControlMethod.VBR\@」類似，但將編碼延遲列入考量。"
```

Comment:
This change is unrelated and should also be handled on Crowdin (https://crowdin.com/project/obs-studio). I've fixed this particular issue, so the next translation update fixes the escape character.